File: loops/fizzbuzz.py
# River Stanley Fizzbuzz

for x in range(0,51):
    if x%5 ==0 and x%3 ==0:
        print("fizzbuzz")
    elif x%5 ==0:
        print("buzz")
    elif x%3 ==0:
        print("fizz")
    else: 
        print(x)

File: loops/fizzbuzz.c
// River Stanley FIZZBUZZ C


#include <stdio.h>
#include <math.h>

int main(void){
   
    int x;
    for(x=0; x<51; x++){
        if (x%5 ==0 && x%3 ==0){
            printf("fizzbuzz\n");
        }else if (x%5 ==0){
            printf("buzz\n");
        }else if (x%3 ==0){
            printf("fizz\n");
        }else{
            printf("%d\n",x);
        }
    }
return 0;
}